city_scrapers/spiders/example.py

```diff
@@ -0,0 +1,231 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class StlAldermenSpider(CityScrapersSpider):
+    name = "stl_public_service"
+    agency = "St. Louis Board of Public Service"
+    timezone = "America/Chicago"
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+    start_urls = [
+        (
+            "https://www.stlouis-mo.gov/government/departments/public-service/index.cfm"
+        )
+    ]
+
+    def __init__(self, *args, **kwargs):
+        self.agenda_map = defaultdict(list)
+        super().__init__(*args, **kwargs)
+
+    def parse(self, response):
+        self._parse_links(response)
+        yield from self._parse_meetings_page(response)
+
+    def _parse_meetings_page(self, response):
+        urls = [
+            (
+                "https://www.stlouis-mo.gov/events/"
+                "past-meetings.cfm?span=-30&department=332"
+            ),
+            "https://www.stlouis-mo.gov/events/all-public-meetings.cfm?span=30",
+        ]
+        for url in urls:
+            yield scrapy.Request(
+                url=url, method="GET", callback=self._parse_events_page
+            )
+
+    def _parse_events_page(self, response):
+        for url in self._get_event_urls(response):
+            yield scrapy.Request(url, callback=self._parse_event, dont_filter=True)
+
+    def _get_event_urls(self, response):
+        event_urls = response.css("ul.list-group h4 a::attr(href)").getall()
+        event_sponsors = response.css("ul.list-group li span.small::text").getall()
+        urls = []
+        for url, sponsor in zip(event_urls, event_sponsors):
+            if "aldermen" in sponsor.lower() or "aldermanic" in sponsor.lower():
```

Comment:
Here, you should change it to something like `if "public service" in sponsor.lower()`. The current code will only scrape events for the Board of Alderman.